plugins/modules/server.py

```diff
@@ -471,9 +471,9 @@ def _create_server(self):
                 self.result["root_password"] = resp.root_password
                 # Action should take 60 to 90 seconds on average, but can be >10m when creating a
                 # server from a custom images
-                resp.action.wait_until_finished(max_retries=362)  # 362 retries >= 1802 seconds
+                resp.action.wait_until_finished(max_retries=1000)  # 362 retries >= 1802 seconds
                 for action in resp.next_actions:
-                    action.wait_until_finished()
+                    action.wait_until_finished(max_retries=1000)
```

Comment:
```suggestion
                    # Starting the server or attaching to the network might take a few minutes,
                    # depending on the current activity in the project.
                    # This waits up to 30minutes for each action in series, but in the background
                    # the actions are mostly running in parallel, so after the first one the other
                    # actions are usually completed already.
                    action.wait_until_finished(max_retries=362) # 362 retries >= 1802 seconds
```

Please change this to a reasonable and documented amount of retries/wait time.